### PR TITLE
Wrapping and timestamp fixes, updated READMEs

### PR DESCRIPTION
**Pull request by ignotus666**
# Does this need translation?

- [ ] Yes 
- [ *] No 

# Changes

**NOTE**: THIS SHOULD ONLY BE MERGED AFTER #589 IS - OR DISCARD THAT AND DIRECTLY MERGE THIS ONE.

I divided them into 2 PRs to make reviewing easier, but whatever - this one includes all the other PR's changes plus some more.

- Corrected the no-wrap command as the previous one was not working as expected.
- Added command to remove the timestamp line starting with "POT-Creation-Date:" in .po files. This mostly solves the issue of commits being flooded with meaningless edits of this line. Before, this line changed in every single .po file, every time `po4a-update-templates.sh` was run - even if no .po files had been edited. Now, though some editors (namely Poedit) add it back in, it will only happen to the files that are actually edited via translation, vastly reducing the problem. Addresses #521.
- As a consequence of the first point, the list of recommended editors has been reduced to 2 (Poedit and OmegaT), as the rest mess up the line wrapping.
- Made the po4a installation error messages a bit clearer.
- Edited the README and OmegaT instruction files in `_translator-files` to update the instructions regarding the preferred editors.

Leaving this here as I'll be gone until September and will likely not be available in the meantime. The wrapping and timestamp issues were seriously bugging me and I wanted to get them out of the way before leaving.

### CODE REVIEW

File: contribute/en/po4a-Setup-Explained.md
Comment (ann0see) [inline contribute/en/po4a-Setup-Explained.md]:
```suggestion
- Translating these documents into each respective language. As translations take place in special .po files, these need to be converted back into .md format for the website.
```

File: contribute/en/po4a-Setup-Explained.md
Comment (ann0see) [inline contribute/en/po4a-Setup-Explained.md]: Do you really want to refer to "new setup"? This implies that we need to remove the reference to "new" someday…

I‘d mainly see this file as an explanation of the process which can be edited/adapted in future and be linked to for new contributors/maintainers/translators/… who need to know something about the po4a process.

File: contribute/en/po4a-Setup-Explained.md
Comment (ann0see) [inline contribute/en/po4a-Setup-Explained.md]: Maybe give a hint that the user should extract the .zip file and run it either with a local web server or jekyll serve. Just double-clicking on index.html doesn’t work.

**Review by ann0see — APPROVED** [review]
Wow! Thank you very much - especially for the detailed description of the po4a process!
Approving it, although I‘d like to have a review from someone else - especially concerning the deletion of the update date in the po files.

File: contribute/en/po4a-Setup-Explained.md
Comment (ignotus666) [reply contribute/en/po4a-Setup-Explained.md]: I see what you mean. I don't think I'll have much access to computers until September, so feel free to edit stuff. If you want, ping me, I'll have a look and commit the edits myself; whatever.

As for the sed command to delete the timestamp line, I know it's a bit hacky, but I would ask that any objections offer an alternative. It does prevent some 85 lines from being needlessly edited every time the script runs. I thoroughly tested it and there were no negative consequences if that's what you're concerned about.

**Review by ann0see — APPROVED** [review]

**Pull request merged**
